lib/iris/tests/test_plot.py

```diff
@@ -323,6 +323,12 @@ def setUp(self):
             'Temperature')
         self.draw_method = qplt.scatter
 
+    def test_cube_coord(self):
+        pass
+
+    def test_coord_coord(self):
+        pass
```

Comment:
This is a bit odd... It looks like we could either do with some content for these tests, or for them to be removed.